bin/create_image.sh

```diff
@@ -68,13 +68,23 @@ cp -r "${DIR}/../../client" /mnt/tmp/usr/src/client >> "${LOG}" 2>&1
 echo "${3}" > /tmp/profile.json
 mv /tmp/profile.json /mnt/tmp/usr/src/client/honeypot_profile.json
 cp -r "${DIR}/../pipot/services" /mnt/tmp/usr/src/client/pipot >> "${LOG}" 2>&1
+cpuArch=$(python ${DIR}/getCpuArch.py)
```

Comment:
I don't see a `getCpuArch.py` file. Did you forget to add it, or am I overlooking something?

Reply:
Oh right, thanks for the double check.